hs_tracking/views.py

```diff
@@ -4,9 +4,39 @@
 from django.views.generic import TemplateView
 from django.utils.decorators import method_decorator
 from django.contrib.auth.decorators import user_passes_test
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 
 from . import models as hs_tracking
+from .models import Session, Variable
+from .utils import get_std_log_fields
+
+
+class AppLaunch(TemplateView):
```

Comment:
Ah.... I was hoping that it could get moved out to the app that is actually wanting to do the logging. But that doesn't make sense since we won't know when a user clicks the a link otherwise since they are getting sent off-site.

On second thought this makes sense and the it potentially could be abstracted from AppLaunch to a more general case with different off-site redirect URLs included in the params (with proper checking) if there are other similar use cases.